src/components/ui/Input.js

```diff
@@ -26,88 +21,71 @@ const Input = React.forwardRef(
       value,
       onChange,
       onEnter,
-      label,
       icon,
       className,
       prepend,
       append,
       ...props
     },
     ref,
-  ) => {
-    const id = label?.toLowerCase().split(' ').join('-').concat(`-${className}`)
-
-    return (
-      <div className={className}>
-        {label && <label htmlFor={id}>{label}</label>}
-        <div>
-          <AddOn prepend> {prepend} </AddOn>
-          <input
-            value={value}
-            onChange={onChange}
-            onKeyDown={(e) => {
-              e.key === 'Enter' && onEnter?.(e)
-            }}
-            type="text"
-            placeholder={placeholder ?? ''}
-            id={id}
-            ref={ref}
-            {...props}
-          />
-          {icon}
-          <AddOn prepend={false}>{append}</AddOn>
```

Comment:
we don't need to pass `prepend` as a prop explicitly since undefined is falsey

Reply:
^^